jvm/src/main/scala/eval/tile/LazyTile.scala

```diff
@@ -19,6 +19,7 @@ import scala.reflect.ClassTag
 
 
 sealed trait LazyTile extends LazyLogging {
+  def extent: RasterExtent
```

Comment:
This is not appropriate place for an extent. I'm looking but what what is the need for it here?